sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/DoNotHardcodeCredentials.cs

```diff
@@ -135,7 +141,7 @@ private static string FindBannedWords(string variableName, string variableValue)
 
             var bannedWordsFound = variableName
                 .SplitCamelCaseToWords()
-                .Intersect(passwordVariants)
+                .Intersect(this.splitCredentialWords)
```

Comment:
Remove `this.`. I thought the editorconfig you set should suggest not to add that.

Reply:
It is faded out.

Reply:
Then why do we keep it?